source/TestData.cpp

```diff
@@ -0,0 +1,110 @@
+/* Test.cpp
+Copyright (c) 2019 by Peter van der Meer
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "DataFile.h"
+#include "DataNode.h"
+#include "DataWriter.h"
+#include "Files.h"
+#include "TestData.h"
+#include <string>
+
+using namespace std;
+
+
+
+TestData::TestData()
+{
+	dataSetName = "";
```

Comment:
These are again the defaults for the datatypes.

Reply:
I'll remove the unnecessary initialization.